src/Codeception/Lib/Interfaces/DataMapper.php

```diff
@@ -10,4 +10,6 @@ public function seeInRepository($entity, $params = []);
     public function dontSeeInRepository($entity, $params = []);
 
     public function grabFromRepository($entity, $field, $params = []);
+
+    public function grabEntitiesFromRepository($entity, $params = []);
```

Comment:
Please remove this method from the interface. 
Unlike the other methods it doesn't return basic types. Also it specifies to return an Entity which may depend on implementation of specific ORM.